recipe/meta.yaml

```diff
@@ -24,14 +24,14 @@ requirements:
         - python
         - setuptools
         - numpy
-        - cdat-lite
-        - udunits2
+        - netCDF4
+        - cfunits
```

Comment:
This package does not exist on conda-forge. You'll need to submit it first to staged-recipes.